### PR TITLE
Ks sk946 update website and mail moving to ovh translation

### PR DESCRIPTION
**Pull request by Kazarus4**
Translation for all subs are done. Links checked with gitpod (I let en/gb links for Exchange because these guides doesn't exist specifically for subs ASIA AU and other EN subs) Exchange ofer available on all subs (commercial page exist).

Need proof for EN, DE and Latina versions.

### CODE REVIEW

File: pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md [hunk 1]
```diff
@@ -1,185 +1,258 @@
...
+Para importar la copia de seguridad del espacio de almacenamiento FTP de su anterior proveedor, [conéctese al espacio de almacenamiento FTP de su alojamiento de OVHcloud](https://docs.ovh.com/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/) y envíe la copia de seguridad a la carpeta raíz "www" (o a otra carpeta raíz que haya creado previamente).
 
-Al no haber cambiado todavía la configuración del dominio, los nuevos mensajes se seguirán recibiendo en las direcciones de correo que tiene con su actual proveedor. Asimismo, deberá seguir utilizando dichas direcciones para enviar mensajes de correo.
+Le recomendamos que utilice el programa [FileZilla](https://docs.ovh.com/es/hosting/web_hosting_guia_de_uso_de_filezilla/) para transferir su copia de seguridad FTP a su alojamiento.
 
-#### 4. Modificar la configuración del dominio
+Si el archivo de backup está comprimido (zippé), descomprimirlo en una carpeta vacía en su ordenador antes de transferir sus archivos al alojamiento de OVHcloud.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md]: Please re-check translation of "zippé" :-)

File: pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md [hunk 2]
```diff
@@ -1,185 +1,258 @@
...
-|5. Conectar el sitio web a la nueva base de datos|En el archivo de configuración de su sitio web seguirá guardada la información de su anterior base de datos. Edite dicho archivo, presente en su espacio de almacenamiento de OVH, indicando la información de la nueva base de datos en OVH.|
+Le recomendamos que no utilice el `Tipo de servidor`{.action} **POP** en la sección `Cuenta de origen`{.action}. Este protocolo elimina los mensajes de correo de su antiguo servidor para enviarlos al servidor de destino de OVHcloud. No podrá comparar el contenido de la antigua dirección y la nueva dirección de correo electrónico.
 
-Al no haber cambiado todavía la configuración del dominio, el alojamiento utilizado para mostrar el sitio web seguirá siendo el de su actual proveedor, en caso de que la resolución DNS siga activa.
+En la sección `Cuenta de destino`{.action}, introduzca únicamente la dirección de correo electrónico de OVHcloud correspondiente y su contraseña asociada. Dejando el `Tipo de servidor`{.action} como `Hosted by OVH (Autodetect)`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md]:
```suggestion
En la sección `Cuenta de destino`{.action}, introduzca únicamente la dirección de correo electrónico de OVHcloud correspondiente y su contraseña asociada, dejando el `Tipo de servidor`{.action} como `Hosted by OVH (Autodetect)`{.action}.
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md [hunk 3]
```diff
@@ -1,185 +1,258 @@
...
 
-Al no haber cambiado todavía la configuración del dominio, los nuevos mensajes se seguirán recibiendo en las direcciones de correo que tiene con su actual proveedor, en caso de que la resolución DNS siga activa. Asimismo, deberá seguir utilizando dichas direcciones para enviar mensajes de correo.
+> [!primary]
+>
+Para ello, es necesario tener las claves de acceso de todas sus antiguas cuentas de correo y el nombre del servidor de correo de su anterior proveedor.
+>
+> Si sus direcciones de correo estuvieran configuradas en POP sin conservar copias de los mensajes en su antiguo servidor de correo, o si tuviera los emails registrados "localmente" en sus dispositivos, solo podrá realizar la **opción 2**.
+>
+
+**Opción 2**: realice una copia de seguridad del contenido de sus direcciones de correo mediante un cliente de correo (Outlook, Mail de Mac...), reconfigure su cliente de correo e importe la copia de seguridad en su nueva dirección de correo de OVHcloud.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md]: "Mail de Mac", please check if it's actually translated

File: pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md [hunk 4]
```diff
@@ -1,185 +1,258 @@
...
+### Etapa 9: sustituir los servidores DNS activos de su dominio por los de OVHcloud <a name="step9"></a>
+
+La zona DNS preconfigurada durante el [etapa 2](#step2) todavía no se ha aplicado al dominio.
+
+Sustituya los servidores DNS actuales de su dominio por los dos servidores DNS declarados en la zona DNS de OVHcloud.
 
 > [!warning]
 >
-> El tiempo de propagación necesario para que la modificación de los servidores DNS sea plenamente efectiva es de un máximo de 24 a 48 horas.
+> El cambio de los servidores DNS debe realizarse desde el actual agente registrador del dominio y tarda entre **propagación de 24 a 48 horas***.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md]:
```suggestion
> El cambio de los servidores DNS debe realizarse desde el actual agente registrador del dominio y tarda entre **propagación de 24 a 48 horas**.
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md [hunk 5]
```diff
@@ -1,184 +1,258 @@
 ---
 title: 'Migrare un sito e un servizio di posta in OVH'
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md]:
```suggestion
title: 'Migrare un sito e un servizio di posta in OVHcloud'
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md [hunk 6]
```diff
@@ -1,184 +1,258 @@
...
+
+- L'hosting [Personale](https://www.ovhcloud.com/it/web-hosting/personal-offer/) con **10 indirizzi email** "MX Plan"
+- Hosting [Pro](https://www.ovhcloud.com/it/web-hosting/professional-offer/) con **100 indirizzi email** "MX Plan"
+- Hosting [Performance](https://www.ovhcloud.com/it/web-hosting/performance-offer/) con **1000 indirizzi email** "MX Plan". Questa offerta è disponibile in 4 "sottolivello".
+- L'hosting [Cloud Web](https://www.ovhcloud.com/it/web-hosting/cloud-web-offer/) con **200 indirizzi email** "MX Plan". Questa offerta è utilizzata dagli sviluppatori di applicazioni.
+
+Una volta scelta la soluzione di hosting, se non sei ancora cliente OVHcloud, clicca su `Commander`{.action} nelle precedenti pagine commerciali. Segui gli step dell'ordine **senza richiedere il trasferimento del tuo dominio**.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md]: Please check IT translation for "Commander"

File: pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md [hunk 7]
```diff
@@ -1,184 +1,258 @@
...
-|2.3.|Crea un database OVH|Se il tuo sito funziona con un database, è necessario [crearne uno nuovo presso OVH](https://docs.ovh.com/it/hosting/gestisci_un_database_dal_tuo_hosting_condiviso/){.external} dal tuo [Spazio Cliente OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it){.external}.|
-|2.4.|Importa i dati del database|Importa il backup del tuo database attraverso lo Spazio Clienti OVH.|
-|2.5.|Associa il sito al nuovo database|Le informazioni relative al tuo precedente database sono sempre disponibili nel file di configurazione del tuo sito. Sul tuo spazio di storage OVH, modifica questo file e inserisci le informazioni del nuovo database OVH.|
+Per importare il backup dello spazio di storage FTP del tuo precedente provider, [accedi allo spazio di storage FTP del tuo hosting OVHcloud](https://docs.ovh.com/it/hosting/accedere-spazio-storage-ftp-hosting-web/) e trasferisci il backup nella cartella root "ww" (o in un'altra cartella root creata precedentemente).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md]:
```suggestion
Per importare il backup dello spazio di storage FTP del tuo precedente provider, [accedi allo spazio di storage FTP del tuo hosting OVHcloud](https://docs.ovh.com/it/hosting/accedere-spazio-storage-ftp-hosting-web/) e trasferisci il backup nella cartella root "www" (o in un'altra cartella root creata precedentemente).
```
Please check for other similar mistakes

File: pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md [hunk 8]
```diff
@@ -1,184 +1,258 @@
...
+#### Per gli account "Email-Pro":
 
-Questa azione può avere diverse conseguenze:
+- Consulta le nostre guide all'aiuto alla configurazione nelle sezioni `Configurazione su computer` e `Configurazione su smartphone` de [la nostra guida sull'offerta Email-Pro](https://docs.ovh.com/it/emails-pro/).
 
-- **vincolare tecnicamente il tuo dominio alle soluzioni OVH**: il tuo hosting OVH sarà utilizzato per mostrare il tuo sito Web e la ricezione dei nuovi messaggi avverrà sui tuoi indirizzi email creati presso OVH
-- **risolvere un’interruzione del servizio**: se il tuo Registrar ha interrotto la propria configurazione DNS durante il trasferimento del dominio, ciò gli consentirà a quest’ultimo di essere nuovamente raggiungibile.
+#### Per gli account email "Exchange":
+
+- Tutte le nostre guide all'aiuto alla configurazione sono disponibili nelle sezioni `Configurazione Exchange su` e `Configurazione Exchange su smartphone` di [la nostra guida sull'offerta Exchange](https://docs.ovh.com/it/microsoft-collaborative-solutions/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md]: Translation of `Configuration Exchange sur ordinateur` is only partial here. Please check https://docs.ovh.com/it/microsoft-collaborative-solutions/ (we should improve sections BTW)

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pl-pl.md [hunk 9]
```diff
@@ -1,185 +1,258 @@
 ---
 title: Przeniesienie strony WWW i kont e-mail do OVH
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.pl-pl.md]:
```suggestion
title: Przeniesienie strony WWW i kont e-mail do OVHcloud
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md [hunk 10]
```diff
@@ -1,174 +1,258 @@
...
+- O seu domínio **com** os "www", para o seu domínio sem os "www", com um registo do tipo "CNAME".
 
-#### 1 - Contratar um alojamento web da OVH
+**Exemplo**: Para o nome de domínio "domain.tld", o resultado deve ser o seguinte:
 
-Aceda ao site [OVH](https://ovh.pt/){.external} e subscreva um serviço de alojamento web. Atenção! Siga os passos aqui indicados antes de solicitar a transferência de domínio. A instalação do serviço inicia automaticamente após a realização do pagamento. No final da instalação, irá receber um e-mail a confirmar a conclusão do processo.
+![hosting](images/DNSzone.png){.thumbnail}
 
-#### 2 - Transferir o site
+> [!success
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md]:
```suggestion
> [!success]
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md [hunk 11]
```diff
@@ -1,174 +1,258 @@
...
+Se o seu ficheiro de backup estiver comprimido (zipado), descomprima-o numa pasta vazia no seu computador antes de transferir os seus ficheiros para o alojamento OVHcloud.
 
-- **associar o seu domínio aos serviços OVH**: o alojamento OVH será usado para colocar o seu site online e enviar e receber e-mail;
-- **evitar a interrupção do serviço**: se o agente de registo suspender a resolução DNS quando solicitar a transferência de domínio, não haverá problema. O sistema OVH está pronto para garantir o funcionamento do site e dos seus e-mails.
+Para o backup da sua base de dados, [crie uma nova base de dados](https://docs.ovh.com/pt/hosting/criar-base-de-dados/) e depois [importe o seu backup para a sua nova base de dados](https://docs.ovh.com/pt/hosting/partilhado_guia_de_importacao_de_uma_base_de_dados_mysql/).
+
+> [!primary]
+>
+> A OVHcloud oferece servidores de base de dados CloudDB. Se pretender utilizar esta oferta com o seu website, consulte toda a nossa documentação sobre este produto na nossa página dedicada <https://docs.ovh.com/pt/clouddb/>.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md]:
```suggestion
> A OVHcloud oferece servidores de base de dados CloudDB. Se pretender utilizar esta oferta com o seu website, consulte toda a nossa documentação sobre este produto na [nossa página dedicada](https://docs.ovh.com/pt/clouddb/).
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md [hunk 12]
```diff
@@ -1,174 +1,258 @@
...
-|4|Efetuar pagamento|A transferência do domínio e a instalação do alojamento irão iniciar após o pagamento. **Alguns agentes de registo suspendem a resolução DNS do domínio após a receção do pedido de transferência. Neste caso, o seu domínio e todos os serviços associados (site e e-mail) poderão ficar indisponíveis.**|
-|5|Validar a transferência|Este processo varia conforme a extensão do domínio. Por vezes, é necessário validar a transferência. Normalmente, o pedido de validação é enviado por e-mail. Siga os passos indicados para validar a transferência. |
+Para migrar o conteúdo dos endereços antigos, existem duas opções.
 
-#### 2 - Transferir o site
+**Opção 1**: utilize a nossa ferramenta [OVHcloud Mail Migrator (OMM)](https://omm.ovh.net/) {.external} que permite copiar o conteúdo dos endereços de e-mail registados no seu antigo fornecedor para os criados na OVHcloud. Para mais informações, consulte o nosso manual "[Migrar contas de e-mail via OVH Mail Migrator](https://docs.ovh.com/pt/microsoft-collaborative-solutions/exchange-migracao-de-contas-email-ovh-mail-migrator/)".
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md]:
```suggestion
**Opção 1**: utilize a nossa ferramenta [OVHcloud Mail Migrator (OMM)](https://omm.ovh.net/){.external} que permite copiar o conteúdo dos endereços de e-mail registados no seu antigo fornecedor para os criados na OVHcloud. Para mais informações, consulte o nosso manual "[Migrar contas de e-mail via OVH Mail Migrator](https://docs.ovh.com/pt/microsoft-collaborative-solutions/exchange-migracao-de-contas-email-ovh-mail-migrator/)".
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md [hunk 13]
```diff
@@ -1,174 +1,258 @@
...
-|4|Importar dados existentes na base de dados inicial|Importe o backup da base de dados usando [a ferramenta disponível na Área de Cliente](https://docs.ovh.com/pt/hosting/partilhado_guia_de_importacao_de_uma_base_de_dados_mysql/){.external}.|
-|5|Associar o site à nova base de dados|As informações da base de dados inicial estão guardadas no ficheiro de configuração do seu site. No espaço de armazenamento OVH, altere o ficheiro, introduzindo as informações relativas à base de dados OVH.|
+Sugerimos que não utilize o `Tipo de servidor`{.action} **POP** na parte `Conta source`{.action}. Este protocolo apaga os e-mails do seu antigo servidor para os enviar para o servidor OVHcloud de destino. Neste caso, já não poderá comparar o conteúdo do antigo e do novo endereço de e-mail.
 
-A configuração do domínio continua inalterada. Se a resolução DNS ainda estiver ativa, o seu site continua a ser apresentado através do alojamento do operador anterior.
+Para a parte `Conta de destino`{.action}, introduza apenas o endereço de e-mail OVHcloud em questão e a respetiva palavra-passe. Isto permitindo o `Type de servidor`{.action} em `Hosted by OVH (Autodetect)`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md]: Please check that `Type de servidor`{.action} is the correct PT wording

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md [hunk 14]
```diff
@@ -1,174 +1,258 @@
...
+
+Depois de migrar os seus antigos endereços de e-mail para a OVHcloud, reconfigure os seus softwares de e-mail com a ajuda de todos os nossos guias sobre o assumpto.
+
+#### Para as contas de e-mail "MX Plan": 
 
-#### 4 - Alterar a configuração do domínio.
+- Encontre os parâmetros de configuração no nosso guia "[Generalidades sobre os e-mails MX Plan](https://docs.ovh.com/pt/emails/partilhado_generalidades_e-mail_partilhado_ovh/#2-utilizar-o-software-a-sua-escolha)". Além disso, poderá encontrar links para os guias de configuração personalizados para os principais softwares de e-mail.
 
-O seu site e o seu domínio foram transferidos para a OVH, e os endereços de e-mail foram criados no sistema da OVH. Agora é necessário alterar a configuração DNS do seu domínio. Esta ação implica substituir os servidores DNS do outro operador pelos servidores DNS da OVH.
+#### Para as contas "Email-Pro" :
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md]:
```suggestion
#### Para as contas "Email-Pro"
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md [hunk 15]
```diff
@@ -1,174 +1,258 @@
...
 
-A alteração é efetuada através da [Área de Cliente OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt){.external}. Se tiver dúvidas, consulte o guia sobre *[Gestão dos servidores DNS](https://docs.ovh.com/pt/domains/partilhado_generalidades_sobre_os_servidores_dns/){.external}*.
+- Encontre os nossos guias de ajuda à configuração nas secções `Configuration on computador` e 'Configuration no smartphone` de [o nosso manual na oferta Email-Pro](https://docs.ovh.com/pt/emails-pro/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md]:
```suggestion
- Encontre os nossos guias de ajuda à configuração nas secções `Configuration on computador` e `Configuration no smartphone` de [o nosso manual na oferta Email-Pro](https://docs.ovh.com/pt/emails-pro/).
```

File: pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md (hunk 1 above)
Comment (Kazarus4) [reply pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md]: comprimido

Comment (Kazarus4) [reply pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md]: best translation deepl => zip

File: pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md (hunk 3 above)
Comment (Kazarus4) [reply pages/web/hosting/hosting_migrating_to_ovh/guide.es-es.md]: Correo de Mac ?

File: pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md (hunk 13 above)
Comment (Kazarus4) [reply pages/web/hosting/hosting_migrating_to_ovh/guide.pt-pt.md]: tipo de serviço

File: pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md (hunk 6 above)
Comment (Kazarus4) [reply pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md]: Ordine

File: pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md (hunk 8 above)
Comment (Kazarus4) [reply pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md]: Configurazione di Exchange sul computer

Comment (Kazarus4) [reply pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md]: done

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**